Tests/Common/IntegrationTestFixtureBase.cs

```diff
@@ -354,7 +354,7 @@ public async Task UpdateExistingFirmwareUpgradeValues(StationEui stationEui, uin
             if (getDeviceResult == null)
                 throw new InvalidOperationException("Concentrator should exist in IoT Hub");
             var deviceTwin = await registryManager.GetTwinAsync(stationDeviceId);
-            var cupsJson = ((object)deviceTwin.Properties.Desired[BasicsStationConfigurationService.CupsPropertyName]).ToString();
+            var cupsJson = (string)(deviceTwin.Properties.Desired[BasicsStationConfigurationService.CupsPropertyName]).ToString();
```

Comment:
Is this cast needed?

Reply:
Cast is needed because at build time, this is a dynamic object.